liquibase/src/main/resources/db/changelog/db.changelog-master.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<databaseChangeLog
+        xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+        xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog
+                      http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.8.xsd">
+
+    <!-- Include the changeset that creates the systems table -->
+    <include file="db/changelog/db.changelog-0001-create-systems-table.xml"/>
```

Comment:
is there a need for the db.changelog- prefix?

Reply:
- [ ] good remark, is a leftover form the place where they were moved from. will remove